packages/eslint-plugin-jest/src/index.js

```diff
@@ -8,11 +8,33 @@
  * @flow
  */
 
-/* eslint-disable global-require */
-
 'use strict';
 
-module.exports.rules = {
-  'no-exclusive-tests': require('./rules/no-exclusive-tests'),
-  'no-identical-title': require('./rules/no-identical-title'),
+module.exports = {
+  environments: {
```

Comment:
Are there any plans to drop the global API, and just to `import { test, expect } from 'jest'` or something in the future? Maybe even just `import test from jest` and having `t` in callbacks, like tap or ava?